src/openfermion/utils/_lcu_util_test.py

```diff
@@ -0,0 +1,181 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Tests for _lcu_util.py."""
+
+from __future__ import absolute_import
```

Comment:
import in one line? Also, again why do you feel this is necessary? Travis would be fine without it.

Reply:
`numers[i] / denom` would return 0 instead of a proper result without this import.